server.js

```diff
@@ -12,3 +12,258 @@ app.listen(app.get('port'), () => {
   console.log(`${app.locals.title} is running on
     http://localhost:${app.get('port')}`);
 });
+
+app.get(`/api/v1/crops`, (request, response) => {
+  const crops = app.locals.crops;
+
+  if (!crops) {
+    return response.sendStatus(404)
```

Comment:
maybe this shouldn't be 404? its technically the endpoint we want them to go to, maybe worth asking at a stand up